compiler/p/codegen/OMRMemoryReference.cpp

```diff
@@ -1618,7 +1618,7 @@ void OMR::Power::MemoryReference::accessStaticItem(TR::Node *node, TR::SymbolRef
          else if (isStaticField && !ref->isUnresolved())
             {
             TR::Register *reg = _baseRegister = cg->allocateRegister();
-            loadAddressConstant(cg, nodeForSymbol, 1, reg, NULL, false, TR_DataAddress);
+            loadAddressConstant(cg, nodeForSymbol, 1, reg, NULL, false, TR_DataAddress, true);
```

Comment:
I don't think this makes sense here or in this PR. Leaving a note here so I don't forget to remove this or take an appropriate action on this.

Reply:
Yeah it looks like it won't compile. Did you meant to add the `true` as an earlier parameter for `isRelocatable`?

Reply:
To be honest I'm not sure why this is here. I have a separate PR opened to address the changes made inside `MemoryReference::accessStaticItem`: https://github.com/eclipse/omr/pull/4442/files. So I think this might have just escaped my radar when opening the PRs.